### ISSUE TITLE
Chrome extension

### ISSUE THREAD

**Issue by ml242** [abrahamian/shop-do#14]
Is this worth porting? Probably. It could make our build process harder... time will tell?

**Comment by abrahamian** [comment]
Definitely: I want to learn to expose an HTTP API for our Meteor app, to power a Chrome extension or other clients. But I think we should start building it only when we have finished features we want in the Meteor app.

Two thoughts: 

1. Is there any reason not to use React in views for the Chrome extension? Hopefully, we can re-use all our components, but replace getMeteorData-related code with regular ajax calls. 
(Haven't read this yet, but saving it here to reference later) https://www.codementor.io/reactjs/tutorial/building-a-chrome-extension-reactjs-broccoli-sass)

2. I think the Chrome environment gives us access to a basic local data store. If that's the case, should we use it for anything?

I am currently swamped with Rails work, but am looking forward to picking this up again mid-week!
